### ISSUE TITLE
Invalid Fulfilment from payee giving 400-Bad request

### ISSUE THREAD

**Issue by sri-miriyala** [mojaloop/project#813]
**Summary**:
When regression testing on a scenario where when payeefsp sends an invalid fulfilment for the POST /transfers request from payerfsp, it is rejecting with a response code 400 instead of giving an error code and error msg at the payeefsp until the timeout period in switch.
**Severity**:
Medium

**Priority**:
Medium

**Expected Behavior**
payeefsp should get an error code "3100" and error msg "Generic validation error"

**Steps to Reproduce**
  1. Send a POST /transfers request from payerfsp to payeefsp
  2. Send PUT /transfers from payeefsp with invalid fulfilment in the body.
  3. Getting a 400- Badrequest on the postman console instead of 3100 and Generic validation error at payeefsp logs.

**Specifications**
  - Component (if known): Transfers
  - Version:6.3.1
  - Platform: MAC OS
  - Subsystem: Mojaloop
  - Type of testing: Functional
  - Bug found/raised by: Sri

**Notes**:

There is an enhancement for this feature which DA is pending.

**Comment by elnyry** [comment]
Thanks Sri. If **you** think this is a medium, we'll address it after the current madness :-)..

**Comment by elnyry** [comment]
hi @sri-miriyala forgot to ask one thing.. A 400 HTTP response should be acceptable if the 'invalid fulfilment' fails Schema validation..

The expected response that we have in the bug here (3100) will happen (via an error callback) only when the Schema validation itself passes but other validations inside the central services fail.

Please confirm.

**Comment by elnyry** [comment]
Here's the current response with helm 7.4.3 (I suspect its been fixed in 7.4.x)
```
{
    "errorInformation": {
        "errorCode": "3101",
        "errorDescription": "Malformed syntax - Invalid transfer fulfilment description.",
        "extensionList": {
            "extension": [
                {
                    "key": "cause",
                    "value": "FSPIOPError: Invalid transfer fulfilment description.\n    at createFSPIOPError (/opt/ml-api-adapter/node_modules/@mojaloop/centr"
                }
            ]
        }
    }
}
```
